corehq/ex-submodules/casexml/apps/case/mock.py

```diff
@@ -239,10 +239,10 @@ def fmt(value):
                 return ''
             if isinstance(value, datetime):
                 return unicode(format_datetime(value))
-            elif isinstance(value, (basestring, int)):
+            elif isinstance(value, (basestring, int, date)):
```

Comment:
I didn't think that was necessary, as `unicode(value)` should print in the ISO_DATE_FORMAT ('%Y-%m-%d') anyways, and the reason for adding the `format_datetime` above is to handle the timezones on datetimes which dates don't have to deal with.